src/payment/card/stripe.js

```diff
@@ -119,7 +120,23 @@ export default class StripeCardPayment extends Payment {
 
     await this.loadScripts(this.scripts);
 
-    return this._confirmCardPayment(intent);
+    try {
+      const result = await this._confirmCardPayment(intent);
+
+      return result;
+    } finally {
+      await this._resetAsyncPayment(payment.id);
+    }
+  }
+
+  /**
+   * Reset the payment timer to update the payment status faster
+   *
+   * @param {string} id
+   * @returns {Promise<object>}
+   */
+  _resetAsyncPayment(id) {
+    return this.request('put', '/payments', id, { $async_reset: true });
   }
```

Comment:
I think it's better to move this method as public to the payments module to call it for any payment method (`swell.payment.resetAsyncPayment(id)`).